autoendpoint/src/error.rs

```diff
@@ -193,6 +203,9 @@ impl ApiErrorKind {
             self,
             // Ignore common webpush errors
             ApiErrorKind::NoTTL | ApiErrorKind::InvalidEncryption(_) |
+            // Ignore GCM router authentication errors otherwise
+            // we could blow through our quota.
+            ApiErrorKind::Router(RouterError::GCMAuthentication) |
```

Comment:
I think we should keep these Auth errors as a sentry event instead of making them a metric, seems to be more of an "internal" (autoendpoint is misconfigured) event. If you really want a metric though, we need to fix the tag value used below (per my code suggestion).